### PR TITLE
Adding combination fixed bug for FIPS issue

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.16

Issue:

N/A

Link to docs preview:

https://77748--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-openshift-cli-bug-fixes_release-notes

QE review:
- [x] QE has approved this change.

Additional information:

**Adding a combo of two bugs (https://issues.redhat.com/browse/OCPBUGS-23386, https://issues.redhat.com/browse/OCPBUGS-28540) to the bug fixes section.**

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -1306,6 +1306,8 @@ To retain the catalog rebuilding functionality, use `--rebuild-catalog`. However
...
 
+* Previously, when running a cluster with FIPS enabled, you might have received the following error when running the OpenShift CLI (`oc`) on a RHEL 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. With this release, the default OpenShift CLI (`oc`) is compiled with {op-system-base-full} 9 and works properly when running a cluster with FIPS enabled on {op-system-base} 9. Additionally, an OpenShift CLI (`oc`) compiled with {op-system-base} 8 is also provided, which must be used if you are running a cluster with FIPS enabled on {op-system-base} 8. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*], link:https://issues.redhat.com/browse/OCPBUGS-28540[*OCPBUGS-28540*])
```
Comment (jeana-redhat) [inline release_notes/ocp-4-16-release-notes.adoc:1309]:
```suggestion
* Previously, when running a cluster with FIPS enabled, you might have received the following error when running the OpenShift CLI (`oc`) on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. With this release, the default OpenShift CLI (`oc`) is compiled with {op-system-base-full} 9 and works properly when running a cluster with FIPS enabled on {op-system-base} 9. Additionally, an OpenShift CLI (`oc`) compiled with {op-system-base} 8 is also provided, which must be used if you are running a cluster with FIPS enabled on {op-system-base} 8. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*], link:https://issues.redhat.com/browse/OCPBUGS-28540[*OCPBUGS-28540*])
```

Could also use `{oc-first}` if you like. 

I suppose the non-first instances should just be `oc`, but I can understand wanting to use a more normal noun phrase given the context ("the default `oc`" and "an `oc`" read p funny imo :sweat_smile:). Maybe "the default version of `oc`" and "a version of `oc`" would work? :thinking: So perhaps:

```suggestion
* Previously, when running a cluster with FIPS enabled, you might have received the following error when running the {oc-first} on a {op-system-base} 9 system: `FIPS mode is enabled, but the required OpenSSL backend is unavailable`. With this release, the default version of `oc` is compiled with {op-system-base-full} 9 and works properly when running a cluster with FIPS enabled on {op-system-base} 9. Additionally, a version of `oc` compiled with {op-system-base} 8 is also provided, which must be used if you are running a cluster with FIPS enabled on {op-system-base} 8. (link:https://issues.redhat.com/browse/OCPBUGS-23386[*OCPBUGS-23386*], link:https://issues.redhat.com/browse/OCPBUGS-28540[*OCPBUGS-28540*])
```

**Review by jeana-redhat — COMMENTED** [review]
One missed "RHEL" and one thing to consider for `oc` :bow: 

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: release_notes/ocp-4-16-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-16-release-notes.adoc:1309]: Good suggestions, thanks!

**Pull request merged**